recipes/RavenPy/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "ravenpy" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 35181f3a4a4b42b9a8d98e78df8b5df1d6e4ae8fd8310a9826c93d88efe2a312
+
+build:
+  # noarch: python
```

Comment:
If the only reason not to have this as `noarch: python` is due to `raven-hydro` only being available on Linux, it's fine to have this as `noarch: python` and ignore the build failures.

Reply:
I suppose so. I know that eventually we would like to expand to at least include macOS (as we know for a fact it's supported), but we can attempt to add it in later (the supporting `raven-hydro` and `ostrich` binaries just need to have their build configurations set up).

I'll add the changes in. Thanks!